xwiki-platform-core/xwiki-platform-bootstrap/pom.xml

```diff
@@ -0,0 +1,186 @@
+    <xwiki.extension.features>org.webjars:bootstrap</xwiki.extension.features>
+  </properties>
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-resources-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>copy-bootstrap-resources</id>
+            <phase>validate</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${project.build.directory}/bootstrap</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>src/main/</directory>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+          <execution>
+            <id>copy-bootstrap-tests-resources</id>
+            <phase>validate</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${project.build.directory}/bootstrap/js/tests</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>src/tests/</directory>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+          <execution>
+            <!-- Copy the generated bootstrap components to the WebJar folder. -->
+            <id>copy-bootstrap-components</id>
+            <phase>process-resources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${project.build.outputDirectory}/META-INF/resources/webjars/${project.artifactId}/${project.version}</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>${project.build.directory}/bootstrap/dist</directory>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+          <execution>
+            <!-- Copy the less bootstrap components to the WebJar folder. -->
+            <id>copy-bootstrap-less-components</id>
+            <phase>process-resources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${project.build.outputDirectory}/META-INF/resources/webjars/${project.artifactId}/${project.version}/less</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>${project.build.directory}/bootstrap/less</directory>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.xwiki.contrib</groupId>
+        <artifactId>jshint-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>default-lint</id>
+            <phase>none</phase>
```

Comment:
I'd add a comment explaining that we're disabling the JSHint check associated with the WebJar packaging because we use ``frontend-maven-plugin`` below to perform linting.